src/main/java/eu/bbmri_eric/negotiator/database/model/Negotiation.java

```diff
@@ -140,15 +127,45 @@ public void setCurrentState(NegotiationState negotiationState) {
   }
 
   public void setStateForResource(String resourceId, NegotiationResourceState state) {
```

Comment:
please add javadoc for public methods

Reply:
Fixed in ffd04f6a2618411ceddbaaf038bd371a7633ad6f